app/views/users/show.html.erb

```diff
@@ -1,2 +1,19 @@
-<h1>Users#show</h1>
-<p>Here is a single User, you are in '/users/id'</p>
\ No newline at end of file
+<% content_for :title, "#{@user.name}'s - Blog" %>
+<section  class="flex flex-col w-[55%]">
+  <%= render "user", user: @user %>
+  <div class = "bio border-2 border-blue-gray" >
+    <h3>Bio</h3>
+    <p><%= @user.bio%></p>
+  </div>
+  
+  <% @user.recent_three.each do |post| %>
+    <%= link_to user_post_path(@user, post) do %>
+      <%= render "posts/post", post: post %>
+    <%end%>
+  <%end%>
+
+  <%= link_to user_posts_path(@user), class: 'self-center' do %>
+  <button type='button' class= "see-all-posts mt-4 button text-white bg-gradient-to-r from-blue-500 to-blue-200 border-2 hover:border-blue-400 hover:text-blue-400 hover:from-white hover:to-white"">See all posts</button>
+  <%end%>
+
+</section>
```

Comment:
- [ ] In the user show page, kindly improve your styling to match the wireframe. The elements should also be centered and spread out to occupy enough horizontal space.

  Your User show page | Wireframe
  :---:|:---:
  ![djdus](https://user-images.githubusercontent.com/36542986/230016607-729439ff-ef95-46dc-b41c-72a4128d3cb0.jpg) | ![iso](https://user-images.githubusercontent.com/36542986/230016598-45e8c92a-2d44-41da-8c5c-79ab810c3e29.jpg)